### ISSUE TITLE
molgenis.org does not log the standard INFO to catalina.out

### ISSUE THREAD

**Issue by Mark-de-Haan** [molgenis/molgenis#3321]

**Comment by mswertz** [comment]
does this hinder us?

**Comment by fdlk** [comment]
Yes, very much.

**Comment by fdlk** [comment]
But how to reproduce?

**Comment by fdlk** [comment]
![logging config](https://cloud.githubusercontent.com/assets/1523838/8473929/4a8ba160-20ad-11e5-90a4-4d7b618928a9.png)

* Molgenis uses native slf4j log classes with an implementation that logs through logback.
* We recently removed the java commons logging jar that logged directly to stdout.
We replaced it with jcl through slf4j libraryj. So now all java commons logging requests (such as those from Spring) also get routed through slf4j and logback.

Our default logback configuration is:
ROOT: WARN
org.molgenis: INFO

So only INFO messages from org.molgenis classes get logged by logback, all other messages need to be WARN or up. This seems correct to me.

But I can imagine that previously INFO messages from Spring et al made it to Standard.out and now they no longer do. If this is the case and we want them back, we should add appenders to the default logback.xml config file for those packages whose INFO messages we're missing.
